src/style.css

```diff
@@ -57,7 +75,7 @@ form input {
 }
 
 .leaders:nth-child(even) {
-  background-color: hsl(182, 95%, 91%);
+  background-color: hsl(175, 73%, 89%);
 }
 
 .btn-submit {
```

Comment:
- Your project looks like the provided Design , :+1: :+1: But there are few changes you need to make before going to the next milestone. 

- [ ] This should have been pointed out by the previous reviewers, According to Microverse, you are meant to remove all the placeholder comments on the readme as they are there to guide you as you customize the readme. I suggest you remove all the placeholder comments here. :smile: :small_airplane: 

- - Screenshots for reference:
![Selection_012](https://github.com/HenaryKhuraijam/Leader-Board/assets/107632780/6c0c233d-33f2-49e1-a61f-0098cc73bcec)
![Selection_013](https://github.com/HenaryKhuraijam/Leader-Board/assets/107632780/16f660b3-6945-40c7-a699-df023ef05f46)

Reply:
Thank you so much for your support. Done